backend/src/main/java/com/ody/mate/controller/MateController.java

```diff
@@ -23,23 +17,15 @@
 public class MateController implements MateControllerSwagger {
 
     private final MeetingService meetingService;
-    private final MateService mateService;
-    private final NotificationService notificationService;
 
     @Override
     @PostMapping("/mates")
     public ResponseEntity<MeetingSaveResponse> save(
             @AuthMember Member member,
             @RequestBody MateSaveRequest mateSaveRequest
     ) {
-        Long meetingId = InviteCodeGenerator.decode(mateSaveRequest.inviteCode());
-        Meeting meeting = meetingService.findById(meetingId);
-        Mate mate = mateService.save(mateSaveRequest, meeting, member);
-
-        notificationService.saveAndSendDepartureReminder(meeting, mate, member.getDeviceToken());
-
-        List<Mate> mates = mateService.findAllByMeetingId(meeting.getId());
+        MeetingSaveResponse meetingSaveResponse = meetingService.findAndSendNotifications(mateSaveRequest, member);
```

Comment:
[제안]

찾고 noti를 보낸다는 구현이 method 명에 드러나 있는 것 같아서 다음과 같은 네이밍은 어떤가요?
```suggestion
        MeetingSaveResponse meetingSaveResponse = meetingService.sendNotifications(mateSaveRequest, member);
```

Reply:
여러 코멘트를 통해 구현을 감추는 메서드명을 제안해줬는데 
메서드의 의미를 잘 전달해 협업 시 오해가 없도록 하기 위해 기존 메서드명을 유지했습니다!